src/test/scala/cromwell/engine/db/slick/SlickDataAccessSpec.scala

```diff
@@ -650,7 +649,6 @@ class SlickDataAccessSpec extends FlatSpec with Matchers with ScalaFutures with
       val workflowInfo = WorkflowDescriptor(workflowId, testSources)
       val task = Task.empty
       val call = new Call(None, callFqn, task, Set.empty[FullyQualifiedName], Map.empty, None)
-      val workflowOutputList = Seq(ReportableSymbol("call.fully.qualified.scope.symbol", WdlArrayType(WdlStringType)))
```

Comment:
I guess this is highlighted as never used? But was this checking that no exceptions were thrown, etc?